targets/TARGET_STM/lp_ticker.c

```diff
@@ -272,9 +298,33 @@ void lp_ticker_set_interrupt(timestamp_t timestamp)
     if (lp_cmpok == false) {
         /* if this is not safe to write, then delay the programing to the
          * time when CMPOK interrupt will trigger */
+        
+        /* If this target timestamp is close to the roll over of the ticker counter
+         * and current tick is also close to the roll over, then we are in danger zone.*/
+        if(((0xFFFF - LP_TIMER_SAFE_GUARD < timestamp) || (timestamp < LP_TIMER_SAFE_GUARD)) && (0xFFFA < last_read_counter))
+        {
```

Comment:
please fix the style in the file (=consistency)

Reply:
Done